Plugin/src/SofaPython3/DataHelper.h

```diff
@@ -221,9 +221,9 @@ void copyScalar(BaseData* a, const AbstractTypeInfo& nfo, pybind11::array_t<T, p
     void* ptr = a->beginEditVoidPtr();
 
     auto r = src.unchecked();
-    for (ssize_t i = 0; i < r.shape(0); i++)
+    for (auto i = 0; i < r.shape(0); i++)
```

Comment:
Do you know the new type of `i`? I want to make sure that you don't introduce a narrow conversion in `i < r.shape(0)`

Reply:
After further research I saw that the ssize_t does not come from the C one, but from pybind11 itself.
The namespace pybind11 was just missing